satpy/readers/seviri_l2_bufr.py

```diff
@@ -98,6 +98,9 @@ def _read_mpef_header(self):
 
     def get_attribute(self, key):
         ''' Get BUFR attributes '''
+        # This function is inefficient as it is looping through the entire
+        # file to get 1 attribute. It causes a problem though if you break
+        # from the file early - dont knowgit  why but investigating - fix later
```

Comment:
typo "knowgit"?!?, please fix to clarify.

Reply:
fixed